tests/Feature/LoginTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class LoginTest extends TestCase
+{
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_login()
+    {
+        $this->artisan('passport:install');
+        $user = User::factory()->create([
```

Comment:
let's prepare variable which will contain array with email and password. We can use this variable in user creating and request.